tasks/common.py

```diff
@@ -206,6 +208,26 @@ def logging_init_file_handler():
     LOG_FILE_HANDLER = fh
 
 
+def config_log_for_task(task_name, uuid):
+    """
+    task_name: name of the task in .freeipa-pr-ci.yaml
+    eg: fedora-26/caless
+    """
+    task_name = task_name.split('/')[1]
```

Comment:
I'm not sure choosing the logger name dynamically based on task name is beneficial. There should always be just a single task in one directory, so it might be better to choose a static name like `job.log`. Then it's easier to find the log and we can avoid some weird collisions (`build.log` -- is it the output of mock or something else?).